docs/en/api/POSIX-API.md

```diff
@@ -73,181 +73,363 @@ For example, running the following commands from the `${ALLUXIO_HOME}` directory
 and that the user that runs the `integration/fuse/bin/alluxio-fuse` script must own the mount point
 and have read and write permissions on it.
+
 Multiple Alluxio FUSE mount points can be created in the same node.
 All the `AlluxioFuse` processes share the same log output at `${ALLUXIO_HOME}/logs/fuse.log`, which is
 useful for troubleshooting when errors happen on operations under the filesystem.
 
-See [alluxio fuse options](#configure-alluxio-fuse-options) and [mount point options](#configure-mount-point-options)
-for more advanced mount configuration.
-
-### Check the Alluxio POSIX API Mounting Status
+See [configuration](#configuration) section for how to improve the Alluxio POSIX API performance especially during training workloads.
 
-To list the mount points; on the node where the file system is mounted run:
+### Check Mount Status
 
+FUSE mount points can be checked via `mount` command:
 ```console
-$ ${ALLUXIO_HOME}/integration/fuse/bin/alluxio-fuse stat
-```
+# Mac
+$ mount
+java@macfuse0 on <mount_point> (macfuse, nodev, nosuid, synchronous, mounted by alluxio)
 
-This outputs the `pid, mount_point, alluxio_path` of all the running Alluxio-FUSE processes.
+# Linux
+$ mount
+alluxio-fuse on /mnt/people type fuse.alluxio-fuse (rw,nosuid,nodev,relatime,user_id=1100,group_id=1100)
+```
 
-For example, the output could be:
+FUSE processes can be found via `jps` or `ps` commands.
 
-```
+Mounted Alluxio path information can be found via Alluxio FUSE script:
+```console
+$ ${ALLUXIO_HOME}/integration/fuse/bin/alluxio-fuse stat
 pid mount_point alluxio_path
 80846 /mnt/people /people
 80847 /mnt/sales  /sales
 ```
 
-### Unmount Alluxio from FUSE
+### Run Operations against the FUSE Mount Point
 
-To unmount a previously mounted Alluxio-FUSE file system, on the node where the file system is
-mounted run:
+After mounting, one can run operations (e.g. shell commands, training) against the local folder:
+```console
+$ cp ${ALLUXIO_HOME}/LICENSE /mnt/people/
+$ ls /mnt/people/LICENSE
+$ cat /mnt/people/LICENSE
+```
+The operations will be translated and executed by the Alluxio system and may be executed on the under storage based on configuration.
 
+Note that unlike Alluxio CLIs which show detailed error messages,
+user operations via Alluxio Fuse mount point will only receive error message pre-defined by FUSE which may not be informative.
+For example, once an error happens, it is common to see:
 ```console
-$ ${ALLUXIO_HOME}/integration/fuse/bin/alluxio-fuse unmount mount_point
+$ ls /mnt/people/LICENSE
+ls: /mnt/people/LICENSE: Input/output error
 ```
 
-This unmounts the file system at the mount point and stops the corresponding Alluxio-FUSE
-process. For example,
+In this case, check Alluxio Fuse logs (located at `${ALLUXIO_HOME}/logs/fuse.log`) for the actual error message.
+For example, the command may fail because unable to connect to the Alluxio master:
```

Comment:
Slightly feeling this example is redundant because it's just an example of an error message, but the point you are making here is "user operations via Alluxio Fuse mount point will only receive error message pre-defined by FUSE which may not be informative.". This example is not illustrating this point but simply a random error.

Reply:
Yeah, changed to a rename operation with clear reason, PTAL